test/utils/initMatchers.ts

```diff
@@ -41,16 +94,16 @@ chai.use((chaiAPI, utils) => {
     const element = utils.flag(this, 'object');
     const id = element.getAttribute('id');
 
-    const virutallyFocusedElement = document.activeElement.getAttribute('aria-activedescendant');
+    const virutallyFocusedElementId = document.activeElement!.getAttribute('aria-activedescendant');
 
     this.assert(
-      virutallyFocusedElement === id,
+      virutallyFocusedElementId === id,
       `expected element to be virtually focused${
         isInKarma() ? '\nexpected #{exp}\nactual: #{act}' : ''
       }`,
       'expected element to NOT to be virtually focused',
       elementToString(document.getElementById(id)),
-      elementToString(virutallyFocusedElement),
+      elementToString(document.getElementById(virutallyFocusedElementId!)),
```

Comment:
@joshwooding Previously we compared an element to an id. If this was intended then we didn't need `elementToString`. What do you think helps more during development?
1. previous behavior
2. comparing two ids
3. comparing two (possible null) elements

Reply:
2. Seems better but unsure

Reply:
See https://github.com/mui-org/material-ui/pull/22185